packages/vision/src/helpers.js

```diff
@@ -74,11 +74,12 @@ var coerceImage = (image, callback) => {
  *   asking for the single feature annotation.
  */
 var _createSingleFeatureMethod = featureValue => {
-  return function(image, options) {
-    return this.annotateImage({
+  return function(image, options = {}, callOptions = undefined) {
```

Comment:
See https://github.com/GoogleCloudPlatform/google-cloud-node/issues/2553#issuecomment-324385621 where I also said that we should not be treating this as a one-off, and coming up with the right solution that works API-wide. I explained in that post other places where `gaxOptions` is used (and how). Maybe we should move the talk back over there?